requirements.txt

```diff
@@ -0,0 +1,2 @@
+click
+hiyapyco
```

Comment:
This isn't that easy. The actual script is executed on the remote host, so you can't even install those via salt in infrastructure repo. And `pip --user` or similar is not really acceptable with its poor integrity verification.